admin_manual/configuration/server/security/password_policy.rst

```diff
@@ -0,0 +1,61 @@
+The Password Policy application enables administrators to define password requirements for user passwords and public links.
+
++------------------------------------------------------------------------------------+----------------+-----------------------+
+| Setting                                                                            | User Passwords | Public Link Passwords |
++====================================================================================+================+=======================+
+| Specify a valid password requirements.                                             | Yes            | Yes                   |
++------------------------------------------------------------------------------------+----------------+-----------------------+
+| Specify a password expiration period.                                              | Yes            | Yes                   |
++------------------------------------------------------------------------------------+----------------+-----------------------+
+| Specify expiration dates for public link shares.                                   | No             | Yes                   |
++------------------------------------------------------------------------------------+----------------+-----------------------+
+| Forced password change on first login.                                             | Yes            | No                    |
++------------------------------------------------------------------------------------+----------------+-----------------------+
+| Disallowing passwords that match a configurable number of previous passwords       | Yes            | Yes                   |
+| (defaults to the previous 3).                                                      |                |                       |
++------------------------------------------------------------------------------------+----------------+-----------------------+
+| Users can be notified a configurable number of days before their password expires. | Yes            | No                    |
++------------------------------------------------------------------------------------+----------------+-----------------------+
+| Users will be notified when their password has expired.                            | Yes            | No                    |
++------------------------------------------------------------------------------------+----------------+-----------------------+
+
+.. note::
+   Active user sessions will **not** end when passwords expire.
+   However, a password change will be forced when the user session expires (e.g., on logout).
+   Token access for app or client authentication is not affected.
+
+.. note::
+   Installing and enabling the application also extends the occ command to support :ref:`the user\:expire-password command <user-expire-password_label>`.
+
+The Security App
+~~~~~~~~~~~~~~~~
+
+.. caution:: The Security and Password Policy apps are mutually exclusive.
```

Comment:
It's not that the apps are mutually exclusive. The brute-force protection part of the 'Security' app can **and should** be used in parallel but configuring password policies in the 'Security' app while having the ones from 'Password Policy' in place can do harm.

Reply:
Ah, then I didn't fully understand you in https://github.com/owncloud/documentation/pull/4285#issuecomment-405849131.